flow360/component/simulation/models/turbulence_quantities.py

```diff
@@ -251,6 +251,43 @@ def TurbulenceQuantities(
     ValueError
         If the TurbulenceQuantities inputs do not represent a valid specification.
 
+
+    Valid combinations
+    ------------------
```

Comment:
```suggestion
    Notes
    -----

    The valid combinations of multiple turbulence quantities is summarized as follows,

```

I would suggest using the existing numpy docstring section keyword like "Notes" as title. The current documentation format is not uniform with other sections: 
![Current](Screenshot_14-3-2025_213524_127.0.0.1.jpeg)